__tests__/app.test.js

```diff
@@ -120,3 +120,64 @@ describe("GET /api/articles", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id/comments", () => {
+  test("get 200 return all comments from an article by article id where the article does have comments", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toHaveLength(11);
+        expect(comments).toBeSorted({ key: "created_at", descending: true });
+        comments.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
+  test("get 200 return null for all elements of a comment object for an article that does not have comments", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toHaveLength(1);
+        expect.objectContaining({
+          comment_id: expect.any(null),
+          votes: expect.any(null),
+          created_at: expect.any(null),
+          author: expect.any(null),
+          body: expect.any(null),
+          article_id: expect.any(Number),
+        });
```

Comment:
this test should just really return an empty array - have a look at your join further down!